docs/whats-new.mdx

```diff
@@ -14,7 +14,8 @@ You can expect this page to update regularly (at least monthly). We'll include t
 
 ## May 2024
 
-- 2024-05-23 - Released [ngrok Agent 3.10.0](/agent/changelog/#ngrok-agent-3100---2024-05-23).
+- 2024-05-24 - [Kubernetes Ingress Controller 0.11.0](https://github.com/ngrok/kubernetes-ingress-controller/blob/main/CHANGELOG.md#0110) released with [Helm chart 0.13.0](https://github.com/ngrok/kubernetes-ingress-controller/blob/main/helm/ingress-controller/CHANGELOG.md#0130).
+- 2024-05-23 - Released [ngrok Agent 3.10.0](/agent/changelog/#ngrok-agent-3100---2024-05-23). This release changes the outbound domain used for checking the Certificate Revocation List, see details in [the changelog](/agent/changelog/#ngrok-agent-3100---2024-05-23).
```

Comment:
```suggestion
- 2024-05-23 - Released [ngrok Agent 3.10.0](/agent/changelog/#ngrok-agent-3100---2024-05-23). This release changes the outbound domain used for checking the Certificate Revocation List which may require changes to firewall or proxy settings. See details in [the changelog](/agent/changelog/#ngrok-agent-3100---2024-05-23).
```

I'd like to make it a little more obvious in the "what's new" page that changes may be required.